### ISSUE TITLE
Lua narrator API

### ISSUE THREAD

**Issue by Aurumaker72** [mkdasher/mupen64-rr-lua-#251]
# Rationale

There is currently no way to create a11y-compliant Lua scripts.

# Proposal

A narrator API should be introduced, which allows Lua scripts to synthesize and play back speech from text using the Windows ScreenReader API.

```lua
-- Makes the narrator say "Hello World".
a11y.say("Hello World")
```

**Comment by Aurumaker72** [comment]
I think this is outside the scope of our project for now.

However, I still think that we should expose some kind of elegant a11y abstraction for visual elements (something like Microsoft Active Accessibility but less convoluted).
